pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -168,9 +168,11 @@ func NewNodePerformance(profile *performancev2.PerformanceProfile) (*tunedv1.Tun
 		}
 	}
 
+	templateArgs[templateRealTimeHint] = "true"
```

Comment:
should I add a comment about the reason to set it to true or it's enough with the api documentation?

Reply:
it's sufficient to explain the rationale in the commit message.